### PR TITLE
Draft of docs translation helper

### PR DESCRIPTION
**Pull request by blinkov**
Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/tools/translate/filter.py [hunk 1]
```diff
@@ -21,20 +23,24 @@ def translate(text):
     if target_language == 'en':
         return text
     else:
+        time.sleep(random.random())
```
Comment (qoega) [inline docs/tools/translate/filter.py]: how long?

**Review by qoega — COMMENTED** [review]
will you add readme?

File: docs/tools/translate/filter.py (hunk 1 above)
Comment (blinkov) [reply docs/tools/translate/filter.py]: 0-1 sec

**Comment by blinkov** [comment]
> will you add readme?

probably, for now it's right behind shebang of shell scripts

**Pull request merged**